<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPortFlowWatch</class>
 <widget class="QWidget" name="NewPortFlowWatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QTabBar::tab{ 
	
	border-image:none;
	color: white; 
	padding: 7px;
	border: 0px solid transparent; 
} 

QTabBar::tab:selected {
	border-image: url(:/images/images/list_left_click.png);
	border-radius:6px;
 	white:red;
	font-size:12px;
	font-weight:bold;
}

QTabBar::tab:!selected {
	border-image: url(:/images/images/list_left_2.png);
	margin-top: 5px;
	border-radius:4px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="currentIndex">
         <number>5</number>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab0">
         <attribute name="title">
          <string> 1~6 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QWidget" name="page0" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab1">
         <attribute name="title">
          <string> 7~12 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QWidget" name="page1" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string> 13~18 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QWidget" name="page2" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab3">
         <attribute name="title">
          <string> 19~24 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QWidget" name="page3" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab4">
         <attribute name="title">
          <string> 25~30 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QWidget" name="page4" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab5">
         <attribute name="title">
          <string> 31~36 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QWidget" name="page5" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab6">
         <attribute name="title">
          <string> 37~42 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QWidget" name="page6" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab7">
         <attribute name="title">
          <string> 43~48 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QWidget" name="page7" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab8">
         <attribute name="title">
          <string> 49~54 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QWidget" name="page8" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab9">
         <attribute name="title">
          <string> 55~60 </string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QWidget" name="page9" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
